public/app/features/variables/query/QueryVariableEditor.tsx

```diff
@@ -190,6 +190,10 @@ export class QueryVariableEditorUnConnected extends PureComponent<Props, State>
                   Optional, if you want to extract part of a series name or metric node segment. Named capture groups
                   can be used to separate the display text and value (
                   <a
+                    className={css`
+                      color: #fff;
+                      text-decoration: underline;
+                    `}
```

Comment:
This needs to be fixed in https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Tooltip/_Tooltip.scss  not inline for this specific link